package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@ukhomeoffice/cop-react-form-renderer",
-  "version": "3.2.6",
+  "version": "3.2.7",
```

Hmmm... I think this constitutes a minor release, not a patch.
```suggestion
  "version": "3.3.0",
```